modules/persistent-storage-hostpath-about.adoc

```diff
@@ -23,13 +23,21 @@ kind: Pod
 metadata:
   name: test-host-mount
 spec:
+  securityContext:
+    runAsNonRoot: true
+    seccompProfile:
+      type: RuntimeDefault
   containers:
   - image: registry.access.redhat.com/ubi9/ubi
     name: test-container
     command: ['sh', '-c', 'sleep 3600']
     volumeMounts:
     - mountPath: /host
       name: host-slash
+    securityContext:
+      allowPrivilegeEscalation: false
+      capabilities:
+        drop: [ALL]
```

Comment:
@bergerhoffer I tested this one because I wanted to see if the existing spacing issue under `containers` affected the pod creation (I think all of the parameters under `containers` should be two spaces over). I am getting:

`Error from server (Forbidden): error when creating "test-host-mount.yaml": pods "test-host-mount" is forbidden: violates PodSecurity "restricted:v1.24": restricted volume types (volume "host-slash" uses restricted volume type "hostPath")
`
Is this part of what you are trying to fix with these PRs?

Reply:
@mburke5678 I believe that this indentation is correct. I've seen it done both this way, and indented two more spaces to the right. I didn't touch indentation when making these updates, because that could open a can of worms. 

As for whether each of these updates actually works, that's what I will rely heavily on QE to verify. I've spot tested some, but not all for all my PRs.

CC @ chao007 @gkarager that this one doesn't work as is right now.

Reply:
CC @chao007 @gkarager that this one doesn't work as is right now.

Reply:
SCC "restricted" does not support volume `hostpath` so. we can meet this error.